ssh-key/tests/public_key.rs

```diff
@@ -1,8 +1,12 @@
 //! SSH public key tests.
 
 use hex_literal::hex;
+#[cfg(feature = "alloc")]
+use ssh_key::public::{Ed25519PublicKey, SkEd25519};
 use ssh_key::{Algorithm, PublicKey};
 use std::collections::HashSet;
+#[cfg(feature = "ecdsa")]
+use {sec1::consts::U32, ssh_key::public::SkEcdsaSha2NistP256};
 
 #[cfg(feature = "ecdsa")]
 use ssh_key::EcdsaCurve;
```

Comment:
These can be grouped. Hopefully this is right per rustfmt:

```suggestion

#[cfg(feature = "ecdsa")]
use {
    sec1::consts::U32,
    ssh_key::{EcdsaCurve, public::SkEcdsaSha2NistP256},
};
```